website_order_invoice_automation/models/sale_order_line.py

```diff
@@ -21,8 +21,7 @@ def _get_to_invoice_qty(self):
             invoice_policy = line.order_id.team_id.invoice_policy or \
                              line.product_id.invoice_policy
             if line.order_id.state in ['sale', 'done']:
-                if invoice_policy == 'order' or line.product_id.type == \
-                        'service':
+                if invoice_policy == 'order':
```

Comment:
@TimLai125 This change is insufficient. Now it lacks consideration on products.

Reply:
@yostashiro I thought we want here to be clean and less complicate?